tests/components/ohme/test_config_flow.py

```diff
@@ -0,0 +1,94 @@
+"""Tests for the config flow."""
+
+from unittest.mock import AsyncMock, MagicMock
+
+from ohme import ApiException, AuthException
+import pytest
+
+from homeassistant.components.ohme.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+
+async def test_config_flow_success(
+    hass: HomeAssistant, mock_setup_entry: AsyncMock, mock_client: MagicMock
+) -> None:
+    """Test config flow."""
+
+    # Initial form load
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+
+    assert result["type"] is FlowResultType.FORM
+    assert not result["errors"]
+
+    # Successful login
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {CONF_EMAIL: "test@example.com", CONF_PASSWORD: "hunter2"},
+    )
+    await hass.async_block_till_done()
+    assert result["type"] is FlowResultType.CREATE_ENTRY
+
+
+@pytest.mark.parametrize(
+    ("test_exception", "expected_error"),
+    [(AuthException, "invalid_auth"), (ApiException, "unknown")],
+)
+async def test_config_flow_fail(
+    hass: HomeAssistant,
+    mock_setup_entry: AsyncMock,
+    mock_client: MagicMock,
+    test_exception: Exception,
+    expected_error: str,
+) -> None:
+    """Test config flow errors."""
+
+    # Initial form load
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+
+    assert result["type"] is FlowResultType.FORM
+    assert not result["errors"]
+
+    # Failed login
+    mock_client.async_login.side_effect = test_exception
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {CONF_EMAIL: "test@example.com", CONF_PASSWORD: "hunter1"},
+    )
+    await hass.async_block_till_done()
+    assert result["type"] is FlowResultType.FORM
+    assert result["errors"] == {"base": expected_error}
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.FORM
    assert result["errors"] == {"base": expected_error}
```
Let's have every config flow test end in either ABORT or CREATE_ENTRY

Reply:
How would I end this test in an ABORT?

Reply:
Not this test in particular, this one would be create entry, but once you've done that, all tests end in either one of those